Kinetic_shape_partition/include/CGAL/KSP_3/Graphcut.h

```diff
@@ -52,7 +52,7 @@ namespace KSR_3 {
     using Delaunay_3 = CGAL::Delaunay_triangulation_3<Kernel>;
```

Comment:
What is the role of the Delaunay  triangulation in the graphcut?

Reply:
In the Graphcut file I also see `double min = 10000000000;`   which should probably be replaced with `std::numeric_limits<double>`

Reply:
Delaunay was a left over and has been removed and the second suggestion was integrated.